common/source/docs/common-escs-and-motors.rst

```diff
@@ -47,6 +47,7 @@ Protocols
     DShot ESCs <common-dshot-escs>
     KDE CAN ESCs <common-kde-can-escs>
     DroneCAN ESCs <common-uavcan-escs>
+    AM32 ESCs <common-am32-escs>
```

Comment:
Hi @Hwurzburg

I've linked it from both the higher level "ESCs and Motors" and the "DroneCAN ESCs" page.  This is the same as for the Currawong and Hobbywing DroneCAN ESCs which are also in both places.

I'd like to give the AM32 ESCs a high profile because it's open source and we (or at least Tridge) is involved with their development

Reply:
To keep things moving along I'm going to remove the link from the top level Motors and ESCs page.. but I need to add the page to a Toc tree somewhere..